main.go

```diff
@@ -114,7 +127,21 @@ func main() {
 		"BuildDate", version.BuildDate,
 	)
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	// Channel that will be notified for the create, Update, delete events on pod object
+	podEventNotificationChannel := make(chan interface{})
+
+	// Custom data store, it should not be accessed directly as the cache could be out of sync
+	// on startup. Must be accessed from the pod controller's data store instead
+	dataStore := cache.NewIndexer(k8s.NSKeyIndexer, k8s.NamespaceIndexer())
+
+	kubeConfig := ctrl.GetConfigOrDie()
+	// Set the API Server QPS and Burst
+	kubeConfig.QPS = float32(custom.DefaultAPIServerQPS)
```

Comment:
let's define DefaultAPIServerQPS and Burst in main.go since it's not specific to custom resource